src/apps/loyalty/containers/forgot_password/index.tsx

```diff
@@ -0,0 +1,125 @@
+import * as React from "react"
```

Comment:
Curious, does it make sense to have these auth components exist outside of the context of the loyalty app? For instance, I am going to need them in the `SaveButton` component, feels like they are generic enough to be re-used.